client.go

```diff
@@ -133,16 +133,9 @@ func (c *Client) Disconnect() error {
 
 func (c *Client) markDisconnected() {
 	c.once.Do(func() {
-		select {
-		case c.client.Disconnected <- true:
-		default:
-		}
```

Comment:
I'm hesitant removing this bit because of situations where someone initiates a disconnect but the server might still have messages for the client:

https://github.com/andreykaipov/goobs/blob/314f152a96bd369e258239fcba015d185cd79977/client.go#L285-L294

Reply:
The rationale is [ of course please let me know if I misunderstood the code :) ]:

From the perspective of the receiver code like this, it does not make any difference if it reads from the channel only once: https://goplay.tools/snippet/bt7F-8gxUGe . So this change would not hurt.

But there is a difference if they read it twice: https://goplay.tools/snippet/52vQxoRDPGY . So this change will fix the behavior in case channel `c.client.Disconnected:` is read multiple times.

Reply:
This is something I didn't know, very cool!